Source/Core/DolphinQt/GameList/GameListModel.cpp

```diff
@@ -140,7 +142,26 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     break;
   case COL_FILE_NAME:
     if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
+    {
+      // If the game is extracted (ex. the file is named main.dol), use the
+      // name of the folder that is two directories up
+      if (game.GetFileName() == "main.dol")
```

Comment:
This check isn't entirely sufficient for checking whether something is an extracted game. See this function: https://github.com/dolphin-emu/dolphin/blob/0a71dda8a0c615e8f387dc12d519cf3415a5ef79/Source/Core/DiscIO/DirectoryBlob.cpp#L261

Reply:
Or just check the BlobType of the GameFile, now that I think about it. That should be the fastest and easiest.